plugins/gcp/resoto_plugin_gcp/resources/compute.py

```diff
@@ -1006,6 +1010,7 @@ class GcpFirewallPolicyRule:
     target_service_accounts: Optional[List[str]] = field(default=None)
 
 
+# TODO Firewall Policies are on org level, parentId is org id or folder id
```

Comment:
They exist on project level and *also* on org level.
We should fetch the one on project level as part of the project collector.

Reply:
yes we should - in another PR 👍🏻